Dockerfile

```
@@ -0,0 +1,16 @@
+FROM docker.io/golang:bookworm AS builder
+
+WORKDIR src
+
+COPY *.go ./
+COPY processors/ ./processors/
+
+RUN go mod init gtfstidy
+RUN go mod tidy
+RUN go build .
+
+FROM docker.io/debian:stable-slim
```

Let's add [OCI annotations](https://github.com/opencontainers/image-spec/blob/v1.1.0/annotations.md), to provide helpful machine-readable infos to a variety of tools.

```suggestion
FROM docker.io/debian:stable-slim

LABEL org.opencontainers.image.title="oxygen-xml"
LABEL org.opencontainers.image.description="A tool for checking, sanitizing and minimizing GTFS feeds."
LABEL org.opencontainers.image.authors="Patrick Brosi <info@patrickbrosi.de>"
LABEL org.opencontainers.image.source="https://github.com/patrickbr/gtfstidy"
```

---

docker/metadata-action actually already adds most (all?) of these labels from the GitHub metadata.

Can you check the pfaedle image (which uses the same process) to see if you're missing something? https://github.com/ad-freiburg/pfaedle/pkgs/container/pfaedle

---

Thanks, I didn't know. Consider this thread irrelevant then.